packages/desktop/views/dashboard/governance/views/DetailsView.svelte

```diff
@@ -21,15 +21,16 @@
     import type { IParticipations } from '@core/governance/interfaces'
     import { openPopup } from '@auxiliary/popup'
     import { ProposalStatus } from '@core/governance/enums'
+    import { proposalsState } from '@core/governance/stores'
     import { networkStatus } from '@core/network'
```

Comment:
Add another layer to the imports, i.e.
```typescript
import { networkStatus } from '@core/network/stores'
```

Reply:
I created a separate task to remove existing barrel files and updating all the imports:
https://github.com/iotaledger/firefly/issues/5433